vision/label/README.md

```diff
@@ -0,0 +1,39 @@
+# Google Cloud Vision API Go example
+
+## Get Dependencies
+
+This sample depends on the following packages. Before getting started, be sure to get them.
+
+```bash
+go get -u golang.org/x/net/context golang.org/x/oauth2/google google.golang.org/api/vision/...
+```
+
+To run tests, getting the following package.
```

Comment:
Actually I'm removing this whole section about getting dependencies.

We don't do that for any of the other samples.